tests/test_generate_index.py

```diff
@@ -155,8 +152,12 @@ def test_generate_release_index(setup_index_tests_integration):
 
     with open(f"{target_worktree}/index.html") as index_file:
         index_content = index_file.readlines()
-    for i in range(0, len(correct_content)):
-        assert correct_content[i].strip() == index_content[i].strip()
+
+    stripped_correct_content = sorted([correct_content[i].strip()
+                                       for i in range(0, len(correct_content))])
+    stripped_index_content = sorted([index_content[i].strip()
+                                     for i in range(0, len(index_content))])
```

Comment:
Is the index even needed?
```suggestion
    stripped_index_content = sorted(
        (c.strip() for c in correct_content)
    )
```